test/check-application

```diff
@@ -653,5 +653,58 @@ class TestNavigator(testlib.MachineCase):
         b.drop_superuser()
         b.wait_in_text(".pf-v5-c-empty-state", "Permission denied")
 
+    def testMultiSelect(self):
+        b = self.browser
+        m = self.machine
+
+        self.restore_dir("/home/admin")
+
+        self.enter_navigator()
+
+        # Check control-clicking
+        m.execute("touch /home/admin/file1 && touch /home/admin/file2")
+        b.click("[data-item='file1']")
```

Comment:
Is this redundant since you ctrl click bellow?

Reply:
the next line in the test. Now i noticed that you select this file1 and the Ctrl selected some other files. Is this on purpose?

Reply:
Yes